### PR TITLE
Added samples of FAQs

### PR DESCRIPTION
**Pull request by Suriya-Balamurugan**
Added UG documentation samples,

- Copy-fonts-to-linux-containers
- Remove-paragraph-with-style

### CODE REVIEW

File: FAQs/Copy-fonts-to-linux-containers/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Copy necessary fonts to Linux containers
+----------------------------------------
+
+In Word to PDF conversion, Essential DocIO uses the fonts which are installed in the corresponding production machine to measure and draw the text. If the font is not available in the production environment, then the alternate font will be used to measure and draw text based on the environment. And so, it is mandatory to install all the fonts used in the Word document in machine to achieve proper preservation.
+
+The fonts present in the location(in Docker container) “/usr/local/share/fonts/” is used for conversion. By default, there will be limited number of fonts available in the container.
+
+You should copy necessary fonts to this location “/usr/local/share/fonts/” before conversion.
+
+How to run the examples
```
Comment (MohanaselvamJothi) [inline FAQs/Copy-fonts-to-linux-containers/README.md:10]: How to run this example

File: FAQs/Copy-fonts-to-linux-containers/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+Copy necessary fonts to Linux containers
+----------------------------------------
+
+In Word to PDF conversion, Essential DocIO uses the fonts which are installed in the corresponding production machine to measure and draw the text. If the font is not available in the production environment, then the alternate font will be used to measure and draw text based on the environment. And so, it is mandatory to install all the fonts used in the Word document in machine to achieve proper preservation.
```
Comment (MohanaselvamJothi) [inline FAQs/Copy-fonts-to-linux-containers/README.md:4]: Remove this para, we will convey only necessary things alone

File: FAQs/Copy-fonts-to-linux-containers/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+The fonts present in the location(in Docker container) “/usr/local/share/fonts/” is used for conversion. By default, there will be limited number of fonts available in the container.
+
+You should copy necessary fonts to this location “/usr/local/share/fonts/” before conversion.
+
+How to run the examples
+-----------------------
+
+*   Download this project to a location in your disk.
+*   Download necessary Microsoft compatible fonts and placed it in the fonts folder of this project.
```
Comment (MohanaselvamJothi) [inline FAQs/Copy-fonts-to-linux-containers/README.md:14]: Create a folder "Fonts" parallelly to .sln file
Download necessary Microsoft compatible fonts and paste into "Fonts" folder.

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: FAQs/Copy-fonts-to-linux-containers/Copy-fonts-to-linux-containers.csproj [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <DockerDefaultTargetOS>Linux</DockerDefaultTargetOS>
+    <DockerfileContext>.</DockerfileContext>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.10.14" />
+    <PackageReference Include="SkiaSharp.NativeAssets.Linux" Version="2.88.0-preview.209" />
```
Comment (MohanaselvamJothi) [inline FAQs/Copy-fonts-to-linux-containers/Copy-fonts-to-linux-containers.csproj:12]: Please use proper SkiaSharp and Harfbuzs NuGets

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: FAQs/Copy-fonts-to-linux-containers/README.md (hunk 1 above)
Comment (Suriya-Balamurugan) [reply FAQs/Copy-fonts-to-linux-containers/README.md]: Addressed.

File: FAQs/Copy-fonts-to-linux-containers/README.md (hunk 2 above)
Comment (Suriya-Balamurugan) [reply FAQs/Copy-fonts-to-linux-containers/README.md]: Addressed.

File: FAQs/Copy-fonts-to-linux-containers/README.md (hunk 3 above)
Comment (Suriya-Balamurugan) [reply FAQs/Copy-fonts-to-linux-containers/README.md]: Addressed.

File: FAQs/Copy-fonts-to-linux-containers/Copy-fonts-to-linux-containers.csproj (hunk 4 above)
Comment (Suriya-Balamurugan) [reply FAQs/Copy-fonts-to-linux-containers/Copy-fonts-to-linux-containers.csproj:14]: Addressed.

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**